### DISCUSSION TITLE
how to use QMCSampler on the TPESampler results

### DISCUSSION THREAD

**Discussion by showkeyjar** [optuna/optuna#4286] [Q&A]
### Don't use GitHub Issues to ask support questions.

I notice that there is new QMCSampler on 3.0,

but my model has been searched use TPE for a long time,

and I notice that QMC is a Quasi Monte Carlo Sampler that generates low-discrepancy sequences.

so, if old TPE search results can see as low-discrepancy sequences.

could I use QMC to search better result on TPE result?

If it can, how to search?

**Comment by xxzzjj222** [comment]
你好！xzj已收到你的邮件，谢谢！
